metadata-service/factories/src/main/java/com/linkedin/metadata/boot/steps/IngestPoliciesStep.java

```diff
@@ -66,10 +70,10 @@ public void execute() throws IOException, URISyntaxException {
         .maxStringLength(maxSize).build());
 
     // 0. Execute preflight check to see whether we need to ingest policies
-    log.info("Ingesting default access policies...");
+    log.info("Ingesting default access policies from: {}...", _policiesResource);
```

Comment:
does this correctly print the _path_ of the resource?

or the contents?

Reply:
The _path_ 

This is the output with default values


```
2023-09-22 10:05:22 2023-09-22 08:05:22,700 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:73 - Ingesting default access policies from: class path resource [boot/policies.json]...
2023-09-22 10:05:22 2023-09-22 08:05:22,710 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:98 - Ingesting default policy with urn urn:li:dataHubPolicy:0
2023-09-22 10:05:22 2023-09-22 08:05:22,725 [main] INFO  c.l.m.entity.EntityServiceImpl:1835 - Ingesting aspect with name dataHubPolicyKey, urn urn:li:dataHubPolicy:0
2023-09-22 10:05:22 2023-09-22 08:05:22,727 [main] INFO  c.l.m.entity.EntityServiceImpl:1835 - Ingesting aspect with name dataHubPolicyInfo, urn urn:li:dataHubPolicy:0
2023-09-22 10:05:22 2023-09-22 08:05:22,746 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:98 - Ingesting default policy with urn urn:li:dataHubPolicy:1
2023-09-22 10:05:22 2023-09-22 08:05:22,762 [main] INFO  c.l.m.entity.EntityServiceImpl:1835 - Ingesting aspect with name dataHubPolicyKey, urn urn:li:dataHubPolicy:1
2023-09-22 10:05:22 2023-09-22 08:05:22,762 [main] INFO  c.l.m.entity.EntityServiceImpl:1835 - Ingesting aspect with name dataHubPolicyInfo, urn urn:li:dataHubPolicy:1
2023-09-22 10:05:22 2023-09-22 08:05:22,810 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:106 - Skipping ingestion of editable policy with urn urn:li:dataHubPolicy:7
2023-09-22 10:05:22 2023-09-22 08:05:22,831 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:106 - Skipping ingestion of editable policy with urn urn:li:dataHubPolicy:view-entity-page-all
2023-09-22 10:05:22 2023-09-22 08:05:22,841 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:106 - Skipping ingestion of editable policy with urn urn:li:dataHubPolicy:view-dataset-sensitive
2023-09-22 10:05:22 2023-09-22 08:05:22,847 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:98 - Ingesting default policy with urn urn:li:dataHubPolicy:admin-platform-policy
2023-09-22 10:05:22 2023-09-22 08:05:22,863 [main] INFO  c.l.m.entity.EntityServiceImpl:1835 - Ingesting aspect with name dataHubPolicyKey, urn urn:li:dataHubPolicy:admin-platform-policy
...
```

And this is the output with `ENV BOOTSTRAP_POLICIES_FILE=file:///datahub/datahub-gms/resources/custom-policies.json`

```

2023-09-22 10:45:27 2023-09-22 08:45:27,027 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:73 - Ingesting default access policies from: URL [file:/datahub/datahub-gms/resources/custom-policies.json]...
2023-09-22 10:45:27 2023-09-22 08:45:27,032 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:98 - Ingesting default policy with urn urn:li:dataHubPolicy:XXX
2023-09-22 10:45:27 2023-09-22 08:45:27,132 [main] WARN  org.apache.velocity.deprecation:62 - configuration key 'resource.loader' has been deprecated in favor of 'resource.loaders'
2023-09-22 10:45:27 2023-09-22 08:45:27,133 [main] WARN  org.apache.velocity.deprecation:62 - configuration key 'class.resource.loader.class' has been deprecated in favor of 'resource.loader.class.class'
2023-09-22 10:45:27 2023-09-22 08:45:27,133 [main] WARN  org.apache.velocity.deprecation:62 - configuration key 'file.resource.loader.class' has been deprecated in favor of 'resource.loader.file.class'
2023-09-22 10:45:27 2023-09-22 08:45:27,133 [main] WARN  org.apache.velocity.deprecation:62 - configuration key 'file.resource.loader.path' has been deprecated in favor of 'resource.loader.file.path'
2023-09-22 10:45:27 2023-09-22 08:45:27,133 [main] WARN  org.apache.velocity.deprecation:62 - configuration key 'runtime.references.strict' has been deprecated in favor of 'runtime.strict_mode.enable'
2023-09-22 10:45:27 2023-09-22 08:45:27,627 [main] INFO  c.l.m.boot.steps.IngestPoliciesStep:115 - Successfully ingested default access policies.
```